tardis/io/util.py

```diff
@@ -87,6 +87,11 @@ def construct_quantity(self, node):
         return quantity_from_str(data)
 
     def mapping_constructor(self, node):
+        mapping_dict = self.construct_mapping(node)
+        if set(mapping_dict.keys()) == {'start', 'stop', 'num'}:
+            return OrderedDict([('linspace', np.linspace(mapping_dict['start'].cgs.value,
+                                                         mapping_dict['stop'].cgs.value,
+                                                         mapping_dict['num']))])
```

Comment:
That seems like a good start, but you've missed two important points:

1. Why do you return an `OrderedDict` containing one `linspace`, instead of directly the `linspace` itself?
2. What about units? You ignore that information currently.